Header header
int16 gain_l
int16 gain_r
int16 exp_l
int16 exp_r
